### PR TITLE
Left to right animation

### PR DESCRIPTION
**Pull request by juanjo-ramos**
Implementing the animation to present the rows with a left to right animation.

### CODE REVIEW

File: components/SlidingRowView/index.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  }
+
+  animateAddOperation() {
+    Animated.parallel([
+      Animated.timing(this.position, {
+        toValue: 0,
+        duration: 50,
+        easing: Easing.ease
+      }),
+      Animated.timing(this.opacity, {
```
Comment (juanjo-ramos) [inline components/SlidingRowView/index.tsx]: Intentionally changing the opacity with `parallel`. We can show you later in your animation to present the cells from top to bottom how to do it with `interpolate`. Thoughts?

File: storybook/stories/LeftToRightRowAnimation/index.tsx [hunk 2]
```diff
@@ -7,14 +7,13 @@
  * @lint-ignore-every XPLATJSCOPYRIGHT1
  */
 
-import React, { Component, RefObject, Ref } from "react";
+import React, { Component, RefObject } from "react";
 import { View, FlatList, SafeAreaView } from "react-native";
 import Button from "../../../components/Button";
-import RowView from "../../../components/AnimatableRowView";
+import RowView from "../../../components/SlidingRowView";
```
Comment (juanjo-ramos) [inline storybook/stories/LeftToRightRowAnimation/index.tsx]: This file is exactly the same as [TopToBottomRowAnimation](https://github.com/juanjo-ramos/react-native-workshop/blob/master/storybook/stories/TopToBottomRowAnimation/index.tsx) but this line.

Could we create an interface that both row classes conform to?

File: components/SlidingRowView/index.tsx (hunk 1 above)
Comment (aichamorro) [reply components/SlidingRowView/index.tsx]: sounds good.

File: storybook/stories/LeftToRightRowAnimation/index.tsx (hunk 2 above)
Comment (aichamorro) [reply storybook/stories/LeftToRightRowAnimation/index.tsx]: I think we could, but I think that may distract a bit the purpose of the workshop if we have to explain it.

Comment (juanjo-ramos) [reply storybook/stories/LeftToRightRowAnimation/index.tsx]: I didn't mean to explain that as part of the workshop. I was more thinking for our benefit. As in leaving the codebase the deal with in a cleaner state. They won't need to work on that file at all.

Comment (juanjo-ramos) [reply storybook/stories/LeftToRightRowAnimation/index.tsx]: Merging as is, we can think about this later

**Pull request merged**